README.md

```diff
@@ -24,15 +24,30 @@ On Windows, the following dependencies are needed for building:
 
 * **git for windows** (https://git-for-windows.github.io/) The `git` command should be available on your `%PATH%`. Make sure that the `cmd` directory in the git for windows installation directory is on the `%PATH%` and *not* its `bin` directory, otherwise mingw32 will complain about `sh.exe` being on the path.
 * **CMake** (http://www.cmake.org/) Once CMake is installed make sure it is available on your `%PATH%`. Check this by running `cmake --version` in the Windows console.
-* **MinGW-W64** >= 4.9.04 (http://mingw-w64.org/doku.php) Once installed, its `bin` directory should be available on your `%PATH%`. Test this by running `mingw32-make --version` in the Windows console.
-* **Python** 3.4 (http://python.org/, note that using Python 3.5 is currently untested on Windows)
-* **NumPy** from http://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy - make sure to get the NON-MKL version!
-* **SciPy** from http://www.lfd.uci.edu/~gohlke/pythonlibs/#scipy
+* **MinGW-W64** >= 5.3.0 (http://mingw-w64.org/doku.php) Once installed, its `bin` directory should be available on your `%PATH%`. Test this by running `mingw32-make --version` in the Windows console.
+* **Python** 3.5 (http://python.org/, note that the version "3.5" is hardcoded across this project)
+* **NumPy** (http://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy)
+  You'll need to download a .whl from this site for Python 3.5 and install it via `pip install <your_whl_file>.whl`.
+* **SciPy** (http://www.lfd.uci.edu/~gohlke/pythonlibs/#scipy)
+  You'll need to download a .whl from this site for Python 3.5 and install it via `pip install <your_whl_file>.whl`.
+* **Protobuf** (https://pypi.python.org/pypi/protobuf/3.1.0.post1)
```

Comment:
Oh, yes, we are building our binding using the protobuf tools, but don't need the protobuf python binding, isn't it?